src/services/state-persistor/state-persistor-layer.mapper.ts

```diff
@@ -46,20 +70,14 @@ class StorageLayerMapper {
     )
   }
 
-  bgLayerIdToBgLayer(bgLayerIdText: string | null): Layer | null {
+  bgLayerNameToBgLayer(bgLayerName: string | null): Layer | null {
     const themes = useThemes()
-    return bgLayerIdText
-      ? (themes.findBgLayerByName(bgLayerIdText) as Layer)
-      : null
+    return bgLayerName ? (themes.findBgLayerByName(bgLayerName) as Layer) : null
   }
 
-  bgLayerTobgLayerId(layer: Layer | undefined) {
+  bgLayerTobgLayerName(layer: Layer | undefined) {
```

Comment:
Not of real importance, but the second 'b' of 'bg' in capital would be more consistent / easier to read `bgLayerToBgLayerName`.

Reply:
Ok, updated.